src/components/caption-list/captionList.tsx

```diff
@@ -110,6 +110,7 @@ export class CaptionList extends Component<Props> {
           const captionProps = this._getCaptionProps(captionData);
           return (
             <Caption
+              data-testid={`caption ${index}`}
```

Comment:
`data-testid` attribute should be 100% uniq. Let's add some prefix to be sure page won't have data-testid conflicts with other libraries.
like:
```
data-testid={`transcript_caption ${index}`}
```
etc